src/runtime.js

```diff
@@ -42,13 +42,13 @@ class FetchEvent extends Event {
 
 window.FetchEvent = FetchEvent;
 
-export async function serve(addr) {
+export async function serve(addr = 'localhost:8080') {
```

Comment:
this won't be necessary as the defaults are already defined in `deployctl.ts`

Reply:
Does prepending `localhost` to `:8080` work?
https://github.com/denoland/deployctl/blob/68527b44374aa7e398a14d3f6a1667a5ba074a4a/deployctl.ts#L54

Reply:
unfortunately no, because the output of `Deno.listen` returns the IP address behind "localhost" so we need to have this `hostname || "localhost"` logic before `Deno.listen` and keep it in a variable that we can interpolate for the output on the console.